domain/src/Access/DomainAccessCheck.php

```diff
@@ -56,7 +56,7 @@ public function checkPath($path) {
    * {@inheritdoc}
    */
   public function access(AccountInterface $account) {
-    $domain = $this->domainNegotiator->negotiateActiveDomain();
+    $domain = $this->domainNegotiator->getActiveDomain();
```

Comment:
This has already been fixed. Is your fork up-to-date?

https://github.com/agentrickard/domain/blob/8.x-1.x/domain/src/DomainNegotiator.php#L116